src/actions/posts.js

```diff
@@ -1143,6 +1153,97 @@ export function moveHistoryIndexForward(index) {
     };
 }
 
+export function handleNewPost(msg) {
+    return async (dispatch, getState) => {
+        const state = getState();
+        const currentUserId = getCurrentUserId(state);
+        const post = JSON.parse(msg.data.post);
+        const myChannelMember = getMyChannelMemberSelector(state, post.channel_id);
+        let websocketMessageProps = {};
+        if (msg) {
```

Comment:
This check seems redundant: if `msg` is falsy, then we'll have excepted on line 1160 trying to access `msg.data.post`. Or should we be checking this earlier?

Reply:
god catch, that's what you get when doing a blind copy/paste

Reply:
updated